### ISSUE TITLE
Change circle size of lineChart?

### ISSUE THREAD

**Issue by JuanCBermudez** [PhilJay/MPAndroidChart#1376]
I would like to know if it is possible to change the size of only ONE circle of my line chart when highlighted. 

Thanks :)

**Comment by PhilJay** [comment]
use a different dataset for that circle and change the size there

**Comment by coolscott** [comment]
![device-2016-01-19](https://cloud.githubusercontent.com/assets/1555062/12415139/dea80c92-bed4-11e5-8af5-c546f40876c6.png)


It's seen in figure 1-(1), figure 1-(2)
<br>
Originally I use 2 datasets in the upper chart and 1 dataset in the lower one,
to highlight item with another circle color, I add 1 dataset in upper and lower one.
The lower one pass, but upper chart with the last item goes wrong,
in LineChartRenderer::drawLinear() because entryTo will be as the same as entryFrom.
<br>
...
Entry entryFrom = dataSet.getEntryForXIndex((mMinX < 0) ? 0 : mMinX, DataSet.Rounding.DOWN);
Entry entryTo = dataSet.getEntryForXIndex(mMaxX, DataSet.Rounding.UP);
...
<br> 
As figure 2, in case of the entry number is 1 in each original dataset :
Unless I skip calling ChartData.checkLegal(),  
I got the thrown java.lang.IllegalArgumentException: 
One or more of the DataSet Entry arrays are longer than the x-values array of this ChartData object
at com.github.mikephil.charting.data.ChartData.checkLegal()
<br> 
Because I used the only extra dataset to make it for the highlight purpose.
<br> 
@PhilJay , any suggestion?  Thank you in advance.
